jina/types/request/data.py

```diff
@@ -127,15 +128,25 @@ def __init__(
                 f'fail to construct a {self.__class__} object from {request}'
             ) from ex
 
-        self._pb_body_wo_data = None
-
     @property
     def is_decompressed(self) -> bool:
+        """
+        Checks if the underlying proto object was already deserialized into a :class:`jina.proto.jina_pb2.DataRequestProto` or
```

Comment:
Maybe explicitly state in the docstring that this does not necessarily mean that the data was decompressed? Maybe there even is a name for this property that would immediately convey that.